docs/usage/customization-guide.md

```diff
@@ -185,6 +185,37 @@ to specify taints in the NodeFeatureRule object.
 > not tolerate the taint are evicted immediately from the node including the
 > nfd-worker pod.
 
+### kubectl plugin
```

Comment:
I'd suggest to have the kubelet plugin as a separate file (and thus a main entry shown on the left sidebar on the docs site, under Usage). Then we can just reference that page from here. WDYT?